Automattic/wp-calypso
# chore: migrate `client/my-sites/*` to `import/order`

#### Background

See #54448

#### Changes proposed in this Pull Request

Migrate `client/my-sites/*` to use `import/order`

#### Testing instructions

N/A

- tests should pass
- changes should only be removing dependency comments and reordering imports
- there should be no eslint warnings related to the `import/order` rule (there can be others)

This is a giant PR but to make it easier to review I created one commit for each subfolder in `client/my-sites`.
